docs/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+AB#1234 <!--- Only if relevant, start with a link to an issue on Azure Dev Ops -->
+
+## Describe your changes
+
+Brief description of your changes - not in-depth because the bulk of the description should be in the task on Dev Ops.
+
+## Checklist before requesting a review
+
+- [x] I have performed a self-review of my code
```

Comment:
I'd like to leave these open by default, to make it a more conscious process for the PR-author.

Over time we can evaluate if all of these items are still relevant, or that we can remove/change/add some.

We could even pull some 'analytics' from them... To see how many of them get checked/changed over time. :D